local_costmap:
    global_frame: odom
    robot_base_frame: base_link
    plugins:
        - {name: rolling_layer, type: "rolling_layer::RollingLayer"}
    publish_frequency: 5.0
    footprint: [[-0.4318,0.3048],[0.4318,0.3048],[0.4318,-0.3048],[-0.4318,-0.3048]]
    width: 20
    height: 20
    origin_x: -10
    origin_y: -10
    update_frequency: 10
    track_unknown_space: false
    unknown_cost_value: 30
    static_map: false
    rolling_window: true
    resolution: 0.1
rolling_layer:
    topic: "/move_base_flex/global_costmap/costmap"
